test/conformance/resources_test.go

```diff
@@ -69,43 +73,7 @@ func TestCustomResourcesLimits(t *testing.T) {
 	}
 	domain := route.Status.Domain
```

Comment:
You should just be able to replace _ on CreateRunLatestServiceReady with objects, and do objects.Route.Status.Domain (or grab it from Service.Status.Domain).

Reply:
Nice.